app/controllers/catalog_controller.rb

```diff
@@ -378,10 +378,13 @@ class CatalogController < ApplicationController
     # whether the sort is ascending or descending (it must be asc or desc
     # except in the relevancy case). Add the sort: option to configure a
     # custom Blacklight url parameter value separate from the Solr sort fields.
-    config.add_sort_field "score desc, pub_date_ssim desc, title_ssim asc", label: "relevance"
-    config.add_sort_field "pub_date_ssim desc, title_ssim asc", label: "year"
-    config.add_sort_field "author_si asc, title_ssim asc", label: "author"
-    config.add_sort_field "title_ssim asc, pub_date_ssim desc", label: "title"
+    config.add_sort_field "score desc, pub_date_ssim desc, title_ssim asc", label: "Relevance"
+    config.add_sort_field "pub_date_ssim desc, title_ssim asc", label: "Date New to Old"
+    config.add_sort_field "pub_date_ssim asc, title_ssim asc", label: "Date Old to New"
+    config.add_sort_field "author_si asc, title_ssim asc", label: "Author A-Z"
+    config.add_sort_field "author_si desc, title_ssim asc", label: "Author Z-A"
```

Comment:
I don't know that it would really matter or be noticable since it's a sccondary sort field, but you could probably just use title_si in place of the title_ssim for all the sort fields

Reply:
I changed it to use `title_si`, but I didn't notice a huge difference.

Reply:
Yeah, I doubt there will be much difference in most cases, just thought you may as well use the sort field, that way IU can't break anything on you if the other field changes or disappears